spec/controllers/contact_controller_spec.rb

```diff
@@ -48,8 +50,7 @@
   end
   context 'with contact e-mail set' do
     before do
-      AppConfig.first
-        .update_attributes(contact_link_location: 'contact@example.com')
+      allow(@ac).to receive(:contact_link_location) { 'contact@example.com' }
```

Comment:
It definitely reads better; I used the curly braces because they were more concise. I'll update to use `and_return`